### PR TITLE
Add hardware labels information

### PR DESCRIPTION
**Pull request by Itxaka**
We now create some default hardware labels in the inventory. Document those

Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

**Comment by Itxaka** [comment]
This has not been released yet as part of an official release of the operator.

File: docs/hardwarelabels.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+sidebar_label: Hardware Labels
+title: ''
+---
+
+
+By default, the registration of the node will result in the inventory of that node containing some default labels that store data
```
Comment (fgiudici) [inline docs/hardwarelabels.md:7]: Maybe we could add a link to the MachineInventory too, something like:
```suggestion
By default, the registration of the node will result in the [inventory](inventory-management.md#machineinventory) of that node containing some default labels that store data
```

File: docs/hardwarelabels.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+relative to the node hardware for easy identification and selection via a [MachineSelector](machineinventoryselectortemplate-reference.md)
```
Comment (fgiudici) [inline docs/hardwarelabels.md:8]: Nit: missing full stop at the end of sentence
```suggestion
relative to the node hardware for easy identification and selection via a [MachineSelector](machineinventoryselectortemplate-reference.md).
```

File: docs/hardwarelabels.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+sidebar_label: Hardware Labels
+title: ''
+---
+
+
```
Comment (fgiudici) [inline docs/hardwarelabels.md:6]: Missing title... maybe something like:
```suggestion
## Hardware Labels
```

**Review by fgiudici — APPROVED** [review]
just found few nits... nice, all the tables! Looks good!

**Comment by Itxaka** [comment]
@nunix do I merge this? Or do we wait until there is versioning? I dont want to push to the docs something that its not available on stable :=)

File: docs/hardwarelabels.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+---
+sidebar_label: Hardware Labels
+title: ''
+---
+
+## Hardware Labels
+By default, the registration of the node will result in the [inventory](inventory-management.md#machineinventory) of that node containing some default labels that store data
+relative to the node hardware for easy identification and selection via a [MachineSelector](machineinventoryselectortemplate-reference.md).
```
Comment (nunix) [inline docs/hardwarelabels.md:8]:
```suggestion
When a node is registered, the data is stored in the [inventory](inventory-management.md#machineinventory) with default labels and values relative to the node hardware.

This data is used for easy identification and selection via a [MachineSelector](machineinventoryselectortemplate-reference.md).
```

**Review by nunix — COMMENTED** [review]
Rephrased the first sentence.

**Pull request merged**